dolphinscheduler-task-plugin/dolphinscheduler-task-spark/src/main/java/org/apache/dolphinscheduler/plugin/task/spark/SparkArgsUtils.java

```diff
@@ -17,19 +17,37 @@
 
 package org.apache.dolphinscheduler.plugin.task.spark;
 
+import org.apache.dolphinscheduler.plugin.task.api.TaskConstants;
+import org.apache.dolphinscheduler.plugin.task.api.TaskExecutionContext;
 import org.apache.dolphinscheduler.plugin.task.api.model.ResourceInfo;
 import org.apache.dolphinscheduler.plugin.task.api.utils.ArgsUtils;
+import org.apache.dolphinscheduler.plugin.task.api.utils.OSUtils;
 import org.apache.dolphinscheduler.spi.utils.StringUtils;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.nio.file.attribute.FileAttribute;
+import java.nio.file.attribute.PosixFilePermission;
+import java.nio.file.attribute.PosixFilePermissions;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
+
+import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.RWXR_XR_X;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * spark args utils
+ * add main logical for support Spark command
  */
 public class SparkArgsUtils {
 
-    private static final String SPARK_CLUSTER = "cluster";
+    private static final Logger logger = LoggerFactory.getLogger(String.format(TaskConstants.TASK_LOG_LOGGER_NAME_FORMAT, SparkArgsUtils.class));
```

Comment:
Maybe we should reuse SparkTask's logger instead of redefining one. So that we can unify the log output.

Reply:
SparkArgsUtils and AbstractTaskExecutor have no inheritance relationship. If you use the same logger, you need to move SparkArgsUtils to SparkTask, and you can define only one logger. Do you think this method is OK?

Reply:
I'm not quite sure about that. @ruanwenjun @zhuangchong Do you have time to take a look at this PR?

Reply:
@sq-q Don't get me wrong. I'm just not familiar about this part. Please wait patiently for others to reply.

Reply:
No, I think this method is also feasible, so I will change it first. If I find it inappropriate in the future, I will change it back.